plugins/amd-gpu/tests/amd-apu.json

```diff
@@ -6,7 +6,7 @@
       "emulation-file": "@enumeration_datadir@/amd-apu-setup.json",
       "components": [
         {
-          "version": "023.010.001.017.000001",
+          "version": "001",
```

Comment:
if this is a number, it should be `1` not `001`, no?

Reply:
Yeah once I added a `convert_version()` callback that is correct!